### DISCUSSION TITLE
SoundCloud integration?

### DISCUSSION THREAD

**Discussion by codegenerator-cz** [Shabinder/SpotiFlyer#157] [Ideas]
Hello there, first of all, great app, good work!

Have you been thinking about soundcloud (maybe mixcloud too) integration? 

Last thing is little request, I am missing some guide in readme how to contribute translations, could you please clarify that a little bit more please?

keep up good work :)

**Comment by Shabinder** [comment]
w8, I will update in a day or two about contribution guidelines, probably this weekend. Will inform you on same. :+1:
